prime-router/src/test/kotlin/fhirengine/azure/FHIRConverterIntegrationTests.kt

```diff
@@ -237,17 +237,16 @@ class FHIRConverterIntegrationTests {
             assertThat(reportAndBundles).transform { pairs -> pairs.map { it.second } }.each {
                 it.matchesPredicate { bytes ->
                     val invalidHL7Result = CompareData().compare(
-                        bytes.inputStream(),
                         cleanHL7RecordConverted.byteInputStream(),
-
+                        bytes.inputStream(),
```

Comment:
The inputs for these tests were in reverse order, which made the output of the comparison utility less useful for finding the issue.